addon/templates/components/frost-object-browser.hbs

```diff
@@ -19,12 +19,9 @@
     <div class="tools">
       <div class="sort"/>
       <div class="pagination">
```

Comment:
Can we include the pagination div in the slot so that it's not present if you don't use pagination?